src/libopensc/card-openpgp.c

```diff
@@ -3131,9 +3136,11 @@ pgp_store_key(sc_card_t *card, sc_cardctl_openpgp_keystore_info_t *key_info)
 		memset(&pubkey, 0, sizeof(pubkey));
 		pubkey.key_id = key_info->key_id;
 		pubkey.algorithm = key_info->algorithm;
-		if (key_info->u.ec.ecpoint && key_info->u.ec.ecpoint_len){
-			pubkey.u.ec.ecpoint = key_info->u.ec.ecpoint;
-			pubkey.u.ec.ecpoint_len = key_info->u.ec.ecpoint_len;
+		if (key_info->u.ec.ecpointQ && key_info->u.ec.ecpointQ_len){
+			pubkey.u.ec.ecpoint = key_info->u.ec.ecpointQ;
+			pubkey.u.ec.ecpoint_len = key_info->u.ec.ecpointQ_len;
+			pubkey.u.ec.oid = key_info->u.ec.oid;
+			pubkey.u.ec.oid_len = key_info->u.ec.oid_len;
 		}
 		else
```

Comment:
clang-format suggested the following change:


```
-		}
-		else
+		} else
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-openpgp.c -->